docs/glossary.md

```diff
@@ -0,0 +1,361 @@
+
+#### Distributed ledger
+
+A cryptographically secure ledger, composed of nodes in a network, where each node holds a copy of the same ledger. 
+
+
+## E
+
+---
+
+#### Expired ticket
+
+Tickets that reached the end of their window without being called to vote - these can be revoked, but do not grant a reward.
+
+
+## F
+
+---
+
+#### Faucet
+
+A mechanism for obtaining free (testnet) tokens.
+
+
+## H
+
+---
+
+#### Hash
+
+The output of a cryptographic hashing function that produces a fixed-size and unique value from variable-size input. 
+
+#### Hash function
+
+A cryptographic function that produces a fixed-size and unique hash value from variable-size transaction input. Decred's Proof-of-Work uses the [BLAKE-256](https://docs.decred.org/research/blake-256-hash-function/) hashing function. 
+
+#### Hashrate
+
+The number of hashes per second computed by miners on the network.
+
+#### Hybrid PoW/PoS
+
+In Decred, a [hybrid PoW/PoS system](https://docs.decred.org/research/hybrid-design/) is used, whereby blocks mined by PoW miners must be approved by ticket holders, which are pseudorandomly selected from the ticket pool. This provides a check on PoW miners and increases the overall cost of attacking the network. When a block is approved, 60% of the block reward goes to the PoW miner, 30% goes to the holders of  tickets called to vote (approve blocks), and 10% goes to the Decred Treasury to fund the project.
+
+
+## I
+
+---
+
+#### Immature ticket
+
+Once tickets are mined they are immature for 256 blocks (about 20 hours) and cannot be called to vote until after this period elapses.
+
+#### Inflation
+
+Increase in the [available supply](https://docs.decred.org/advanced/inflation/) of Decred as new DCR is minted into existence through the block reward.
+
+
+## I
+
+---
+
+#### Live ticket
+
+Tickets that are waiting to be called.
+
+
+## M
+
+---
+
+#### Mainnet
+
+The proper Decred network - a term used to differentiate from testnet.
+
+#### Mempool
+
+Transactions waiting to be mined.
+
+#### Miners
+
+Miners help secure the network via Proof-of-Work (PoW) mining. Miners recieve block rewards and transaction fees for successfully validating blocks. 
+
+#### Mining Pool
+
+A group of miners who share (pool) their computational resources to mine DCR. When a member of a mining pool successfully mines a block, the rewards are typically split among all pool members in proportion to the hashpower they contribute to the pool. 
+
+#### Missed ticket
+
+Tickets that have been called but did not receive a reward. This can happen if a ticket is called to vote and does not respond. This can also happen if a ticket is called to vote, responds and broadcasts their vote to the network, but a miner does not include the vote in the following block. 
```

Comment:
This would read better if it referred to voting wallets watching for called tickets and voting when possible, rather than tickets themselves responding to votes.

Reply:
Agreed. Reworded to:

#### Missed ticket

Tickets that have been called but did not receive a reward. This can happen if a ticket is called to vote, but the wallet that bought the ticket does not respond. This can also happen if the wallet does respond and broadcasts its vote to the network, but a miner does not include their vote in the following block.